src/app/home/home.component.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+
+import { HomeComponent } from './home.component';
+import { OktaAuth } from '@okta/okta-auth-js';
+import { RouterTestingModule } from '@angular/router/testing';
+import { OKTA_CONFIG, OktaAuthModule } from '@okta/okta-angular';
+
+describe('HomeComponent', () => {
+  let component: HomeComponent;
+  let fixture: ComponentFixture<HomeComponent>;
+  const config = {
+    issuer: 'https://not-real.okta.com',
+    clientId: 'fake-client-id',
+    redirectUri: 'http://localhost:4200'
+  };
+  const oktaAuth = new OktaAuth(config);
```

Comment:
I'm happy to make a follow up PR for mocks so we can provide a template for testing that devs can run with.

Reply:
That'd be awesome! Please keep in mind you'll want to update both the `main` and `widget` branches.